### ISSUE TITLE
Feature request: An option to keep name of arguments in functions

### ISSUE THREAD

**Issue by tarnishablec** [terser/terser#622]
Feature request

sometimes function arg names need to be kept so that arg names can be got in runtime.
I use it to parse function's arguments to web component's propnames.

Complete CLI command or ```minify()``` options used

Version

```terser``` 4.6.6

```terser``` input
```ts
function f(name){
    return name
}
```

```terser``` output
```ts
function f(name){return name}
```

thank you!

**Comment by fabiosantoscode** [comment]
Sounds like something that would be useful. But if it comes out that functions written in the new binary representation for JavaScript doesn't have proper `toString` methods I'd have to not support this to avoid confusion.

**Comment by fabiosantoscode** [comment]
Issue in the proposal: https://github.com/tc39/proposal-binary-ast/issues/76

**Comment by tarnishablec** [comment]
thank you for explanation, going to find another way to do it

**Comment by fabiosantoscode** [comment]
I think this should be kept open!

Inspecting argument names may be very useful.

**Comment by antonkatz** [comment]
@fabiosantoscode when you say "to avoid confusion", could you elaborate? I'm not quite following what the confusion would be.

**Comment by fabiosantoscode** [comment]
@antonkatz there would be confusion if you were expecting your function arguments to be present, but they are gone after distributing your JavaScript code through the binary AST proposal.

**Comment by antonkatz** [comment]
Right... my use case would not work anyways. It depends on `.toString()` call, and since that would not return anything meaningful anymore, it would break either way.

**Comment by zhouyc98** [comment]
I have the same request. 
In my case, I have many functions in a class. I only want to mangle some of them (e.g., functions naming ends with a "_") while keeping the others (including their args' name) unchanged. 
I want to make the "_"-ends functions "private" since they don't need users' intention, and let the others be kept (including function name and function args' name) as "public" for users.
